Add homeassistant.reload_all service (#26184)
home-assistant/home-assistant.github.io: source/_integrations/homeassistant.markdown

Reads the configuration files and checks them for correctness, but **does not** load them into Home Assistant. Creates a persistent notification and log entry if errors are found.

### Service `homeassistant.reload_all`

Reload all YAML configuration that can be reloaded without restarting Home Assistant.

It calls the `reload` service on all domains that have it available. Additionally,
it reloads the core configuration (equivalent to calling
`homeassistant.reload_core_config`) and themes (`frontend.reload_themes`).

> Prior to reloading, it run a basic configuration check. If that fails, the reload

Reviewer: Change to "Prior to reloading, a basic configuration check is performed."

Author: ```suggestion
Prior to reloading, a basic configuration check is performed. If that fails, the reload
```

Author: Thanks!
